src/stories/Motion.mdx

```diff
@@ -0,0 +1,39 @@
+import { Meta, Story, Canvas, Controls, Stories } from '@storybook/blocks';
+import * as MotionStories from './Motion.stories.js';
+
+<Meta of={MotionStories} />
+
+# Motion
+
+[Figma Design Source](https://www.figma.com/file/5TqtPa7KWfhJbQv6ELnbqf/Foundation?node-id=72%3A17784&mode=dev)
+
+## Example
+
+<Stories />
+
+### Controls
+
+<Controls of={MotionStories.Transition} />
+
+## Usage
```

Comment:
I think we need to add a couple more sections here:

1. Add a usage section for using the CSS Variables directly, without the SASS Mixin.
2. Add a section with a table or similar that lists all of the variables/tokens and their values.

Reply:
Done